module/Activity/src/Activity/Service/Activity.php

```diff
@@ -282,11 +283,19 @@ protected function generateActivity(array $params, $user, $organ, $dutch, $engli
         $em->persist($activity);
         $em->flush();
 
-        //$this->getEmailService()->sendActivityCreationEmail($activity);
+        // Send an email when a new Activity was created.
         $this->getEmailService()->sendEmail('activity_creation', 'email/activity',
             'Nieuwe activiteit aangemaakt op de GEWIS website | New activity was created on the GEWIS website',
             ['activity' => $activity]);
 
+        // Send email to GEFLITST if user checked checkbox of GEFLITST
+        if ($activity->getRequireGEFLITST()) {
```

Comment:
Doing this here will also send out an email everytime the activity is updated, which probably isn't that handy.